packages/PoppyPrint-Core.package/PPFormatter.class/instance/visitBlockNode..st

```diff
@@ -14,11 +14,15 @@ visitBlockNode: aNode
 		
 		self printBlockComment: aNode.
 		aNode statements do: [:statement |
-			(self isEmptyLineMarker: statement) ifTrue: [self newLine] ifFalse: [
-				self visitNode: statement.
-				statement ~= aNode statements last ifTrue: [
-					stream nextPut: $..
-					self newLine]].
+			(self isEmptyLineMarker: statement)
+				ifTrue: [self newLine]
+				ifFalse: [
+					((self isOptionalEmptyLineMarker: statement) or: [self isOptionalNewLineMarker: statement])
```

Comment:
reading this for the third time, maybe consider creating a `shouldSkipStatement:` method or similar?

Reply:
read the other one and it's slightly different. so we don't see the need to create a new method. Only if you think this explicit method should be extracted?